semgrep-core/src/parsing/tree_sitter/Parse_rust_tree_sitter.ml

```diff
@@ -2480,7 +2480,7 @@ and map_return_expression (env : env) (x : CST.return_expression) : G.expr =
 
 and map_scoped_identifier_name (env : env)
     ((v1, v2, v3) : CST.scoped_identifier) : G.name =
-  let _either_optTODO =
+  let _either_opt =
```

Comment:
This could probably get renamed to something more descriptive (and without an underscore in front).